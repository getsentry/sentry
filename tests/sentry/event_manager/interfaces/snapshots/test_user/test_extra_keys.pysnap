---
created: '2019-03-14T17:12:36.812874Z'
creator: sentry
source: tests/sentry/event_manager/interfaces/test_user.py
---
errors: null
to_json:
  data:
    extra1: foo
    extra2: bar
