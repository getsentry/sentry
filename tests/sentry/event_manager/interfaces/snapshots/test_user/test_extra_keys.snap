errors: null
to_json:
  data: {extra1: foo, extra2: bar}
